styles/style.css

```diff
@@ -0,0 +1,111 @@
+/*.grid-container {
+  display: inline-grid;
+  grid-template: 1fr 1fr 1fr 1fr 1fr/ 1fr ;
+  grid-gap: 15px;
+}*/
+
+nav ul li{
+  display:inline-block;
+  width: 400px;
+  height: 250px
+  padding: 5px;
+  text-align: center;
+  list-style: none;
+}
+section {
+  text-align: center;
+  padding: 15px;
+}
+section h1{
+  font-size: 3em;
+  text-align: center;
+  font-family: sans-serif;
+  padding: 15px;
+  background-color: #DCCFEC
+}
+section sub {
```

Comment:
Some center alignment issues in this container. I'm not sure how `display: inline-block` works but I believe you can achieve a centered look using `flex` and `justify-content`.